tests/unit/cn/kblock_builder_test.c

```diff
@@ -255,6 +255,7 @@ MTF_DEFINE_UTEST_PRE(test, t_kbb_create1, test_setup)
     kbb_destroy(kbb);
 }
 
+#ifndef __clang__
```

Comment:
What are the issues with clang here?

Reply:
Nested functions that I don't want to deal with right now...